### DISCUSSION TITLE
set_chat_permissions method permission parameter

### DISCUSSION THREAD

**Discussion by C0mputist** [eternnoir/pyTelegramBotAPI#1503] [Q&A]
Hello there,

Could you please explain how to pass permission parameter to set_chat_permissions method.

I have tried below code but it returns an error.

`bot.set_chat_permissions(message.chat.id, {"can_send_messages":False})`

and this one as well

`bot.set_chat_permissions(message.chat.id, ("can_send_messages", False))`

**Comment by coder2020official** [comment]
You should pass types.ChatPermissions object.
https://core.telegram.org/bots/api#setchatpermissions
